go-example/metrics/gooteltest/cmd/oteltester/main.go

```diff
@@ -208,12 +204,18 @@ func main() {
 	if err != nil {
 		log.Fatalf("Error opening config file: %v", err)
 	}
+
+	prefix := "cum_"
+	if config.AggregationTemporalitySelector == "delta" {
+		prefix = "delta_"
+	}
+
```

Comment:
Consider making a function out of this such as metricPrefix(config.AggregationTemporalitySelector). Then you can call it from within registerMetricObservers and registerHistograms. See my other comment below.